src/helpers/data_page.ts

```diff
@@ -1,10 +1,16 @@
 declare const PCore;
 
 // eslint-disable-next-line import/prefer-default-export
-export const getDataPage = (dataPageName, context) => {
+export const getDataPage = (dataPageName, parameters, context) => {
+  let dataViewParams;
+  if(parameters){
+    dataViewParams = {
+      'dataViewParameters': parameters
+    };
+  }
   return new Promise((resolve, reject) => {
     PCore.getDataApiUtils()
-      .getData(dataPageName, null, context)
+      .getData(dataPageName, { ...dataViewParams }, context)
```

Comment:
can't we send dataViewParams directly instead of spread operation?

Reply:
Yes, we can. I'll update. Thanks!